src/hotspot/share/gc/parallel/psScavenge.cpp

```diff
@@ -234,7 +234,7 @@ class PSThreadRootsTaskClosure : public ThreadClosure {
 };
 
 class ScavengeRootsTask : public WorkerTask {
-  StrongRootsScope _strong_roots_scope; // needed for Threads::possibly_parallel_threads_do
+  Threads::ThreadsClaimTokenScope _threads_claim_token_scope; // needed for Threads::possibly_parallel_threads_do
```

Comment:
I wonder if this class could just be named ClaimTokenScope so that we don't need to see the word Threads twice?
```
Threads::ThreadsClaimTokenScope
```
vs
```
Threads::ClaimTokenScope
```
OTOH, the function is named `change_thread_claim_token`. Maybe the second reviewer could weigh in on this.

Reply:
Moved to the end of `threads.hpp`.